### PR TITLE
Unnumbered eob references

### PR DESCRIPTION
**Pull request by wilkus27**
Issue: https://github.com/openstax/kitchen/issues/266

### CODE REVIEW

File: lib/kitchen/composite_page_element.rb [hunk 1]
```diff
@@ -43,13 +43,21 @@ def is_index?
...
     end
 
+    # Returns true if this page is a book section reference
+    #
+    # @return [Boolean]
+    #
+    def is_section_reference?
+      has_class?('os-references-container')
+    end
+
```
Comment (wilkus27) [inline lib/kitchen/composite_page_element.rb:61]: I checked and class `reference` is used in books that contain references in form of citation, similar to footnotes:

image

While class `references` is used when book contain a whole section in Introduction page:

image

Comment (brittany-johnson) [reply lib/kitchen/composite_page_element.rb:61]: The one letter difference is so unfortunate 😂

Comment (brittany-johnson) [reply lib/kitchen/composite_page_element.rb:61]: Maybe it would be good to add the difference that you explained in this comment to the file

**Review by brittany-johnson — APPROVED** [review]
Looks good!

**Review by kswanson33 — APPROVED** [review]

File: lib/kitchen/composite_page_element.rb (hunk 1 above)
Comment (wilkus27) [reply lib/kitchen/composite_page_element.rb:70]: Yes, sure :slightly_smiling_face: It will be less confusing in the future. I added needed comment.

**Pull request merged**